transformer_engine/jax/csrc/extensions/normalization.cpp

```diff
@@ -36,6 +36,9 @@ pybind11::tuple GetNormForwardWorkspaceSizes(size_t batch_size, size_t hidden_si
     output_tensor.set_columnwise_data(static_cast<void *>(&temp), out_dtype, input_shape);
   }
 
+  // Enable norm with cuDNN instead of TE kernels
+  nvte_enable_cudnn_norm_fwd(true);
```

Comment:
Is there a reason we are forcing the cuDNN backend? It seems like something that should be controlled with `NVTE_NORM_FWD_USE_CUDNN`. If cuDNN has better perf, it would be better to change the default to `NVTE_NORM_FWD_USE_CUDNN=1` rather than hard-coding it only for JAX.